.github/actions/build-and-deploy-backend-function/action.yml

```diff
@@ -52,7 +52,8 @@ runs:
       run: |
         cat src/main/resources/${{ inputs.module }}/envvars.yml | \
             sed "s|_GCLOUD_PROJECT_|${{ env.GCLOUD_PROJECT }}|g" | \
-            sed "s|_DATASET_|${{ inputs.dataset }}|g" > ${{ inputs.envvars_yml }}
+            sed "s|_GOOGLE_BUCKET_|${{ inputs.bucket }}|g" | \
+            sed "s|_BQ_DATASET_|${{ inputs.dataset }}|g" > ${{ inputs.envvars_yml }}
```

Comment:
This seems unnecessarily complicated to me.  Do the values for bucket, dataset, and module change?  Why not just set them in envvars.yml and get rid of the sed substitution.  Do any of those values qualify as secrets?  Let's make it as easy as possible for future developers (including us) to understand what's going on.

Reply:
I have simplified the action and restructured the `envvars.yml` values in `<module>/<project>` directory. The `envvars.yml` is also used to populate the `env` properties during deployment.